.github/workflows/pr_limit_reminders.yml

```
@@ -0,0 +1,50 @@
+name: PR Limit Reminders
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

```suggestion
    types: [opened, reopened, ready_for_review]
```
This is when a draft PR is marked "ready for review" - so the number of "open" PRs increases.